code/src/components/Album.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+
+
+const Album = (albumTitle) => {
```

Comment:
Since this component only contains the album title maybe the component could be named AlbumTitle?